File: pascal/moyenne.pas
program MoyenneDeTroisNombre;

var nombre1, nombre2, nombre3: integer;
var resultat: real;

begin
   writeln(' Entrez le premier nombre ');
   readln(nombre1);

   writeln(' Entrez le second nombre ');
   readln(nombre2);

   writeln(' Entrez le dernier nombre ');
   readln(nombre3);

   resultat := (nombre1 + nombre2 + nombre3) / 3;
   writeln(' La moyenne de ', nombre1, ' ; ', nombre2, ' et ', nombre3, ' est : ',resultat:0:2); 
end.

File: python/moyenne.py
def moyenne(a, b, c) :
    return (a + b + c) / 3

N1 = float(input("Entrer un nombre : "))
N2 = float(input("Entrer un second nombre : "))
N3 = float(input("Entrer un dernier nombre : "))    

S = moyenne(N1, N2, N3)

print(f"La moyenne de  {N1}, {N2} et {N3} est {S:.2f}")